cmd/fyne_demo/screens/widget.go

```diff
@@ -274,6 +274,34 @@ func stopProgress() {
 	endProgress <- struct{}{}
 }
 
+func makeListTab() fyne.CanvasObject {
+	var data []string
+	for i := 0; i < 1000; i++ {
+		data = append(data, fmt.Sprintf("Test Item %d", i))
+	}
+
+	icon := widget.NewIcon(nil)
+	label := widget.NewLabel("Select An Item From The List")
+	hbox := fyne.NewContainerWithLayout(layout.NewHBoxLayout(), icon, label)
```

Comment:
Could try using a Split Container to see how the list copes with many resizes as the divider is moved

Reply:
This should be resolved.